automation_tools/satellite6/upgrade/satellite.py

```diff
@@ -66,7 +66,7 @@ def satellite6_setup(os_version):
     # Set satellite hostname in fabric environment
     env['satellite_host'] = sat_host
     logger.info('Satellite {} is ready for Upgrade!'.format(sat_host))
-    if os.environ.get('RUN_EXISTANCE_TESTS'):
+    if os.environ.get('RUN_EXISTANCE_TESTS').lower() == 'true':
```

Comment:
This environment variable is from jenkins job parameters and hence it will be either false or true. This will be always defined.